### PR TITLE
some trivial compiler/PVS-Studio warning fixes

### PR DESCRIPTION
**Pull request by neusdan**

### CODE REVIEW

File: src/build-data/buildh.in [hunk 1]
```diff
@@ -22,7 +22,7 @@
 
 #define BOTAN_DISTRIBUTION_INFO "%{distribution_info}"
 
-#define BOTAN_INSTALL_PREFIX "%{prefix}"
+#define BOTAN_INSTALL_PREFIX R"(%{prefix})"
```
Comment (webmaster128) [inline src/build-data/buildh.in]: Great! I was thinking a long time about how to fix this issue. (Change necessary for Windows when backslash is used, e.g. `%{prefix}` = C:\Botan)

File: src/lib/stream/rc4/rc4.cpp [hunk 2]
```diff
@@ -111,6 +111,6 @@ void RC4::clear()
 /*
 * RC4 Constructor
 */
-RC4::RC4(size_t s) : SKIP(s), X(0), Y(0) {}
+RC4::RC4(size_t s) : SKIP(s), X(0), Y(0), state(), buffer(), position(0) {}
```
Comment (webmaster128) [inline src/lib/stream/rc4/rc4.cpp]: I think `state` and `buffer` are default initialized (i.e. empty vector) anyway, are they really causing warnings? Can we move `byte X = 0; byte Y = 0; size_t position = 0` to the header? Those initializations are not specific for this constructor but useful default.

Comment (neusdan) [inline src/lib/stream/rc4/rc4.cpp]: You're right. Explicit initialization of the vectors is not necessary. Moved initialization of `X`, `Y` and `position` to the header.

**Comment by webmaster128** [comment]
LGTM. The only question is if unused test data in `unit_ecc.cpp` should be removed or commented out.

**Comment by randombit** [comment]
Thanks! The unused test data can go, I think it was to test compressed vs uncompressed points but that is better handled elsewhere anyway.

**Pull request merged**